front/src/modules/timesStops/TimesStopsOutput.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+
+import { compact, keyBy } from 'lodash';
+import { useSelector } from 'react-redux';
+
+import { useSimulationResults } from 'applications/operationalStudies/hooks';
+import type {
+  PathPropertiesFormatted,
+  SimulationResponseSuccess,
+} from 'applications/operationalStudies/types';
+import type {
+  SimulationResponse,
+  TrainScheduleBase,
+  TrainScheduleResult,
+} from 'common/api/generatedEditoastApi';
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import { useOsrdConfSelectors } from 'common/osrdContext';
+import { formatSuggestedOperationalPoints } from 'modules/pathfinding/utils';
+import type { OperationalPointWithTimeAndSpeed } from 'modules/trainschedule/components/DriverTrainScheduleV2/types';
+import { interpolateValue } from 'modules/trainschedule/components/DriverTrainScheduleV2/utils';
+import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
+import { useFormattedOperationalPoints } from 'modules/trainschedule/hooks';
+import { getSelectedTrainId } from 'reducers/osrdsimulation/selectors';
+import { stripDecimalDigits } from 'utils/numbers';
+import { ISO8601Duration2sec, secToHoursString } from 'utils/timeManipulation';
+
+import TimesStops from './TimesStops';
+import { TableType } from './types';
+
+type TimesStopsOutputProps = {
+  train: TrainScheduleBase;
+  simulatedTrain: SimulationResponseSuccess;
+  pathProperties: PathPropertiesFormatted;
+  infraId: number;
+};
+
+const computeRealMargin = (
+  trainSimulation: SimulationResponse | undefined,
+  op: OperationalPointWithTimeAndSpeed
+) => {
+  if (trainSimulation?.status === 'success') {
+    const exactBaseTime = interpolateValue(trainSimulation.base, op?.position * 1000, 'times');
+    const exactProvisionalTime = interpolateValue(
+      trainSimulation.provisional,
+      op?.position * 1000,
+      'times'
+    );
+    const realMargin = (100 * (exactProvisionalTime - exactBaseTime)) / exactBaseTime;
```

Comment:
When `exactBaseTime` === 0, it can return NaN / Infinity.
Maybe add this ?

```suggestion
    const realMargin = (100 * (exactProvisionalTime - exactBaseTime)) / exactBaseTime || 0;
```